impl/src/main/java/io/kroxylicious/testing/kafka/invm/InVMKafkaCluster.java

```diff
@@ -21,30 +22,33 @@
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+import org.apache.kafka.common.KafkaException;
 import org.apache.kafka.common.utils.Time;
 import org.apache.zookeeper.server.ServerCnxnFactory;
 import org.apache.zookeeper.server.ZooKeeperServer;
 import org.jetbrains.annotations.NotNull;
 
-import io.kroxylicious.testing.kafka.api.KafkaCluster;
```

Comment:
formatter-maven-plugin is reverting this change.

Reply:
Argh! Will fix. 

Need to have another go at syncing the config